easybuild/easyconfigs/g/gemelli/gemelli-0.0.9-foss-2022a.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'gemelli'
+version = '0.0.9'
+
+homepage = 'https://github.com/biocore/gemelli'
+description = """
+Gemelli is a tool box for running both Robust Aitchison PCA (RPCA) and
+Compositional Tensor Factorization (CTF) on sparse compositional omics
+datasets.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('scikit-learn', '1.1.2'),
+    ('scikit-bio', '0.5.7'),
+    ('h5py', '3.7.0'),
+]
+
+exts_list = [
+    ('tax2tree', '1.1', {
+        'modulename': 't2t',
+        'checksums': ['f1d28fb99c41bb10cc3c06bfd232891b0ad713f674a5c2ac2991dd99ed512da1'],
+    }),
+    ('iow', '1.0.6', {
+        'modulename': 'bp',
+        'preinstallopts': """sed -i "/'nose /d" setup.py && """,
```

Comment:
Why not include nose and 
```
'runtest': 'nosetests'
```
(or whatever the syntax is, i think this works)

Reply:
nose is not maintained for a long time and not compatible with python3.10
https://github.com/nose-devs/nose/issues/1099